lib/iris/tests/unit/lazy_data/test_array_masked_to_nans.py

```diff
@@ -31,41 +31,45 @@
 
 
 class Test(tests.IrisTest):
-    def test_masked(self):
-        masked_array = ma.masked_array([[1.0, 2.0], [3.0, 4.0]],
-                                       mask=[[0, 1], [0, 0]])
-
-        result = array_masked_to_nans(masked_array).data
-
+    def _common_checks(self, result):
         self.assertIsInstance(result, np.ndarray)
         self.assertFalse(isinstance(result, ma.MaskedArray))
         self.assertFalse(ma.is_masked(result))
```

Comment:
Sure.

FWIW, I inherited these checks. I kept these last two because I wanted to be _absolutely sure_ that my non-masked array really hadn't retained its mask somehow!